### DISCUSSION TITLE
Increase VRam Usage?

### DISCUSSION THREAD

**Discussion by Juugah** [lllyasviel/Fooocus#324] [Q&A]
Is it possible to increase the VRam usage for faster renders? Or does it not work like that.

**Comment by dirkteucher** [comment]
I sometimes run multiple instance of fooocus at the same time manually with a bat file and this as the contents . 

**python launch.py**

This will get your GPU working harder and will allow you to render more images per minute. But I do not know a way to get fooocus to utilize more of the 80% of free resources my GPU has. (I too wish there was a way)

On my 4090 I can run 3 fooocus instances at the same time and have them all rendering at the same time to pump out 50 images total. This gets the vram usage up to about 20gb and the amount of images generated every 10 minutes is almost 3 times higher. So it works. 

I am sure there are better ways to be able to batch out large amounts of images simultaneously and if anyone has any please do post a comment.  

Personally I would prefer to render out a single image faster as that would be far more useful.
